datasets/bkt/bkt.json

```diff
@@ -0,0 +1,367 @@
+{
+  "type": "dataset",
+  "id": "bkt",
+  "title": "bkt",
+  "status": "beschikbaar",
+  "description": null,
+  "version": "0.0.1",
+  "crs": "EPSG:28992",
+  "auth": "FP/MDW",
+  "tables": [
+    {
+      "id": "eigendomsrecht",
+      "type": "table",
+      "schema": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "required": [
+          "schema",
+          "bkBktEigendomsrecht"
```

Comment:
Zou in de required niet gewoon de "id" moeten staan, omdat die via de provenance wordt hernoemd?